packages/api/src/services/actions/lsp.ts

```diff
@@ -36,6 +38,35 @@ export function Handlers(config: Config, appState: Dependencies): Actions {
     async shortAddresses() {
       return Array.from(shortAddresses.values());
     },
+    async listTvls(currency: CurrencySymbol = "usd") {
```

Comment:
these are all the new tvl calls for lsp only. Copied over from the EMP calls for the most part.  These calls are only scoped to lsps.  future work will need to combine emp and lsp tvl